### DISCUSSION TITLE
sage.PlaybackException

### DISCUSSION THREAD

**Discussion by jt4242** [OpenSageTV/Community#115] [General]
Sage has suddenly started acting up on me and I have no idea where to go - Any time I try to play a file, it stops and I get this on my screen

There was a Playback Error in playback.  Details:
sage.PlaybackException
OK

If anyone could give me any leads, I'd appreciate it so much.   I had an accident recently and have had some memory results which doesn't help in the already difficult process of getting SageTV help!!

**Comment by wayner9** [comment]
Did you try restarting/rebooting your SageTV server and also the client or extender that you are playing the file back on.  This sometimes fixes these type of problems.

**Comment by jt4242** [comment]
I did, and then I also restored an old image back up but it happened later too. This time I tried unplugging the HD 300 and plugging it back in and that seems to have helped.  I just don't know why it's happened.  although the fan I have sitting under the HD 300 doesn't seem to be working so well so maybe the machine is just getting too hot?thank you for your suggestion and support !On Feb 20, 2024, at 1:12 PM, wayner9 ***@***.***> wrote:﻿
Did you try restarting/rebooting your SageTV server and also the client or extender that you are playing the file back on.  This sometimes fixes these type of problems.

**Reply by wayner9** [reply]
It is good to have some activity here.  This place never gained traction after the old forum went away and now the traffic has slowed to a crawl.

**Reply by jt4242** [reply]
I agree - and I especially miss it when something like this happens and I've no good forum to search!  Thank you again for responding to my question - I'll let you know if the unplugging of the HD300 helps if it happens again - I'm also putting a stronger fan underneath the box.  The thought of not having use of Sage with all the files I add had me so depressed!!!

**Comment by TGsell1** [comment]
Yes! All keep monitoring and helping each other.  Sent from my iPhoneOn Feb 20, 2024, at 6:01 PM, wayner9 ***@***.***> wrote:﻿
It is good to have some activity here.  This place never gained traction after the old forum went away and now the traffic has slowed to a crawl.

**Comment by jimcgreen** [comment]
I am having a similar problem after a hard drive failure crashed my Win 10 machine. After a new hard drive, Win 10 Pro reinstall and then SageTV 9.2.14.1043 install. I am getting a "(-3,0x80070005) There was a problem accessing the filesystem for capture" error when previewing a channel. I also notice that I can't view Live TV.
All of the source setups for my 8 tuners has a line that says "currently being used for Live TV: false" and I can't change it to true.

**Reply by JustFred51** [reply]
Sounds like Sage isn't able to write to the Recording folder(s).  Maybe a filesystem permissions issue?  
Are you running Sage as a service and trying to write to a mapped drive letter?  If so, verify that the Sage service Log On tab isn't configured to log on as Local System account.  If this doesn't resolve the issue, probably best to start a new discussion thread.

**Reply by jimcgreen** [reply]
That was it! I needed to run Sage as administrator. Now it's working! Thanks!
